packages/portal-proto/src/components/Table/VerticalTable.tsx

```diff
@@ -538,6 +538,7 @@ function VerticalTable<TData>({
                   dataLength={tableData?.length}
                   status={status}
                   pageSize={pageSize}
+                  customDataTestID="xl-hidden-text-showing-count"
```

Comment:
[BECAUSE OF THIS ISSUE](https://cdis.slack.com/archives/C03NVF9B05C/p1714594533391169)